package.json

```diff
@@ -22,6 +22,7 @@
     "@types/react-dom": "^18.2.8",
```

Comment:
check package.json . we must not have any types packages in dependencies, please move them to dev dependencies ;-)